Metadata/AmazonMetadataBuilder.php

```diff
@@ -13,7 +13,7 @@
 
 use Aws\S3\Enum\CannedAcl;
 use Aws\S3\Enum\Storage;
-use Guzzle\Http\Mimetypes;
+use Core23\MimeUtil\MimeUtil;
```

Comment:
Actually I think having a library which supports both a static list and PHP's `finfo` would be great.